modules/services/keynav.nix

```diff
@@ -0,0 +1,37 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.keynav;
+
+in {
+  options.services.keynav = {
+
+    enable = mkEnableOption "keynav";
+
+    package = mkOption {
```

Comment:
I think you can omit this? It seems to me that one rarely would like to change this package.

Reply:
Removed.